types/string/README.md

```diff
@@ -0,0 +1,14 @@
+# `std::string`
+
+## Fields
+
+ * `[Split]Index{32,64}`
+
+with the corresponding column type for the first (principal) column.
+
+## Entries
+
+1. One character strings, with ascending values
```

Comment:
General comment/question. Was the README intended to be sufficient to know what is in the file?  As is the README(s) are general description are knowing what is really in the file require parsing/reading the write.C (or later the reference file).  Is that the intent?

Reply:
That's a good question and I haven't fully made up my mind yet. I think I see arguments for both ways (basically a full standalone description vs duplication in multiple places), so for now I was lazy and decided not to duplicate. If we decide that we want a full description in the READMEs, I can put the following structure (in a follow-up PR):
1. One character strings, with ascending values:
    * `Index32`: `"a"`
    * `Index64`: `"b"`
    * `SplitIndex32`: `"c"`
    * `SplitIndex64`: `"d"`

but it would literally by the same as in the `write.C`...

Reply:
As discussed on Friday, we will link to the reference files once they are produced. I opened https://github.com/root-project/rntuple-validation/issues/37 to track.